library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -40,10 +40,27 @@ public Reader(ContentResolver resolver, Cursor cursor) {
             mCursor = cursor;
         }
 
-        public DownloadInfo newDownloadInfo(Context context, SystemFacade systemFacade, StorageManager storageManager, DownloadNotifier notifier) {
-            final DownloadInfo info = new DownloadInfo(context, systemFacade, storageManager, notifier);
+        public DownloadInfo newDownloadInfo(
+                Context context,
+                SystemFacade systemFacade,
+                StorageManager storageManager,
+                DownloadNotifier notifier,
+                DownloadClientReadyChecker downloadClientReadyChecker) {
+
+            Fuzz fuzz = new Fuzz();
```

Comment:
Yes I thought of that but because that comes from Google I wanted to maintain the consistency of the name just in case we have to merge back again from them, so we know what is what.

Can rename if we don't really care about merging again in the future?